src/Mod/TechDraw/Gui/QGISVGTemplate.cpp

```diff
@@ -176,70 +188,76 @@ void QGISVGTemplate::createClickHandles()
     // XPath query to select all <text> nodes with "freecad:editable" attribute
     // XPath query to select all <tspan> nodes whose <text> parent
     // has "freecad:editable" attribute
-    query.processItems(QStringLiteral("declare default element namespace \"" SVG_NS_URI "\"; "
+    // this is effectively a loop with each pass using the next <text>/<tspan>
+    query.processItems(QString::fromUtf8("declare default element namespace \"" SVG_NS_URI "\"; "
                                          "declare namespace freecad=\"" FREECAD_SVG_NS_URI "\"; "
                                          "//text[@" FREECAD_ATTR_EDITABLE "]/tspan"),
                        [&](QDomElement& tspan) -> bool {
-        QString fontSizeString = tspan.attribute(QStringLiteral("font-size"));
+
         QDomElement textElement = tspan.parentNode().toElement();
-        QString textAnchorString = textElement.attribute(QStringLiteral("text-anchor"));
-        QString name = textElement.attribute(QString::fromUtf8(FREECAD_ATTR_EDITABLE));
 
         double x = Rez::guiX(
-            textElement.attribute(QStringLiteral("x"), QStringLiteral("0.0")).toDouble());
+            textElement.attribute(QString::fromUtf8("x"), QString::fromUtf8("0.0")).toDouble());
```

Comment:
This looks like a mistake 🤔

Reply:
That's original code, so it has been there for a long time, but now that you point it out, how does this even compile?  QDomElement::attribute is supposed to return a QString and Rez::guiX does not have a version that accepts a QString.

Not near my devel box at the moment.  I'll look deeper later.